.github/workflows/installations.yml

```diff
@@ -22,6 +22,8 @@ jobs:
     runs-on: macos-11
     env:
       plist_secret: ${{ secrets.GHASecretsGPGPassphrase1 }}
+      # TODO(v9): Remove `POD_LIB_LINT_ONLY` after publishing `HeartbeatLoggingTestUtils`.
```

Comment:
I'm not sure we want to publish `HeartbeatLoggingTestUtils`. The test specs don't need to be runnable from `pod spec lint`.

Reply:
Ahh I see, thanks. So `HeartbeatLoggingTestUtils` would only be linted on CI when we're pulling in local sources. 

I added this TODO in a bunch of places so I can go back and revert those TODOs. cc: @ryanwilson

Reply:
Right. SGTM

Reply:
Sorry for not catching that, thanks!

Reply:
Cleaned these up in https://github.com/firebase/firebase-ios-sdk/pull/9554